### ISSUE TITLE
please fix build for kernel-5.15... ( at time 5.15.3 )

### ISSUE THREAD

**Issue by ohquait** [DisplayLink/evdi#321]
* Are you using the latest driver?
    
  * Are you using the latest EVDI version?
    
  * If you are using a DisplayLink device, have you checked 'troubleshooting'
    on DisplayLink's website?
    
  * Is this issue related to evdi/kernel?
    

  * Linux distribution and its version
  * Linux kernel version
  * Xorg version (if used)
  * Desktop environment in use

THANKS

**Comment by dimich-dmb** [comment]
Compilation for 5.15 fails because sources refer to drm_irq.h, and it's removed from kernel sources.

**Comment by ollyollyollyltd** [comment]
+1 for this

Running Manjaro 21.2.0 and kernel 5.15.2
As @dimich-dmb says, the build fails due to `drm_irq.h` missing:

```
In file included from /var/lib/dkms/evdi/1.9.1/build/evdi_gem.c:16:
/var/lib/dkms/evdi/1.9.1/build/evdi_drm_drv.h:23:10: fatal error: drm/drm_irq.h: No such file or directory
   23 | #include <drm/drm_irq.h>
      |          ^~~~~~~~~~~~~~~
compilation terminated.

Full [make.log](https://github.com/DisplayLink/evdi/files/7581090/make.log)

**Comment by displaylink-mlukaszek** [comment]
Thanks for this report, this is already being looked at. I'd close this as a duplicate of #308.
